doc/content/documentation/repl.md

```diff
@@ -3,53 +3,64 @@ title = "REPL"
 weight = 18
 +++
 
-Phel provides a interactive prompt where you can type Phel expressions and see the results. This interactive prompt is called REPL (stands for Read-eval-print loop). A REPL is very helpful to test out small task or to play around with the language its self.
+Phel comes with an interactive prompt. The prompt accepts Phel expressions and directly returns the result. This interactive prompt is called REPL (stands for Read-eval-print loop). A REPL is very helpful to test out small task or to play around with the language its self.
```

Comment:
Missing `s` in "to test out small taskS or..."?

`its self` doesn't exist, should be `itself`.